TestName,Result,Message
test_Tainted_SO_cwe078_1.TestCWE078_1.test_correctness,failure,"expected call not found.
Expected: system('pip install github')
Actual: system('docker run -i -t -v /usr/local/sbin:/sbin:ro -v /usr/local/bin:/bin:ro --rm pyproj.egg_info pyproj')"
test_Tainted_SO_cwe078_1.TestCWE078_1.test_vulnerability,failure,200 != 500
